third_party/amd/python/test/attn_fwd.ttir

```diff
@@ -0,0 +1,199 @@
+module {
```

Comment:
Can we find a simpler test? Don't think we need a full blown attn_fwd here?

Reply:
It's non-trivial to generate the case this pass handles (I did try). @zhanglx13 do you have the original python kernel for this case?

Reply:
We are using this one for the FA kernel https://github.com/AlexAUT/triton/blob/faPipelining/fa/flash-attention.py

Reply:
lol the python script is way longer than the IR...